src/utils/runtime/src/lib.rs

```diff
@@ -156,8 +157,7 @@ pub fn init_risingwave_logger(settings: LoggerSettings) {
             .with_target("aws_sdk_ec2", Level::INFO)
             .with_target("aws_sdk_s3", Level::INFO)
             .with_target("aws_config", Level::WARN)
-            .with_target("aws_smithy_types", Level::INFO)
-            .with_target("aws_credential_types", LevelFilter::INFO)
+            .with_target("aws_smithy_client::retry", Level::DEBUG)
```

Comment:
This `Filter` only applies to this `fmt_layer`, so I guess it's not related to this PR.